packages/jest-haste-map/src/__tests__/index.test.js

```diff
@@ -1201,7 +1201,7 @@ describe('HasteMap', () => {
   });
 
   it('distributes work across workers', () => {
-    const jestWorker = require('jest-worker');
+    const jestWorker = require('jest-worker').Worker;
```

Comment:
Does this make this PR a breaking change?

Reply:
@ljharb Yes it needed to be changed, if there is a better way, i will appreciate some guide

Reply:
The solution is to change the "main" of jest-worker to be untranspiled CJS, and to `require` the transpiled file. That gives you precise control over the exports.

Reply:
@ljharb i dont understand your explanation, can u break it down a bit

Reply:
I'm saying, don't use a babel-transpiled file that uses `export` syntax - use a normal node CJS file that uses `require` and `module.exports`.

Reply:
I don't want the extra file/work that is maintaining a "separate" CJS entrypoint, so the change I'll be accepting is allowing CJS to be `const {Worker} = require('jest-worker');` (i.e. skipping `.default`), not removing `__esModule` from the exported object

Reply:
That's very unfortunate imo; it adds a UX burden for all consumers to avoid a (in my experience) minor maintenance burden.

That said, will ESM consumers also have the same experience as CJS consumers - ie, no default, just a named export?

Reply:
>That said, will ESM consumers also have the same experience as CJS consumers - ie, no default, just a named export?

Yes. Or rather, ESM via TypeScript/Babel etc - native ESM will have `.default.Worker` I believe due to Node's CJS->ESM handling. I haven't verified, though